content/docs/guides/auto-suspend-guide.md

```diff
@@ -7,49 +7,62 @@ updatedOn: '2023-08-09T08:55:53Z'
 
 Neon's [Auto-suspend](/docs/introduction/auto-suspend) feature controls when a Neon compute instance transitions to an `Idle` state (scales to zero) due to inactivity. This guide demonstrates how to configure the _Auto-suspend_ setting in your Neon project.
 
-You can configure the _Auto-suspend_ setting in an existing project by editing a compute endpoint. You can also configure it when you first create a Neon project, which sets the _Auto-suspend_ default for the project. Both methods are described below.
+[Neon Pro plan](/docs/introduction/pro-plan) users can configure the _Auto-suspend_ setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the _Auto-suspend_ setting for a new or existing project sets the project's default Auto-suspend setting, which is used from that point forward when creating compute endpoints.
 
-## Configure Auto-suspend for a compute endpoint
+### Configure the Auto-suspend setting for a new project
 
-[Neon Pro plan](/docs/introduction/pro-plan) users can edit an individual compute endpoint to adjust the _Auto-suspend_ configuration.
+Configuring the _Auto-suspend_ setting for a new project sets the project's default, which is applied to all compute endpoints created from that point forward. You can adjust _Auto-suspend_ default or configure the setting for individual compute endpoints later, as necessary.
 
-To edit a compute endpoint:
+To configure the _Auto-suspend_ default setting at project creation time:
```

Comment:
Consider changing "at project creation time" to "when you first create your project"

Reply:
Fixed. TY.